### ISSUE TITLE
Rename variable class to support compilation as part of an C++ project.

### ISSUE THREAD

**Issue by timhoffm** [jlaurens/synctex#11]
I'm including the synctex sources as part of an C++ project. However, the variable name `class` does not work with g++ because it's a reseved symbol.

*synctex_parser_advanced.h*:
~~~
struct synctex_node_t {
    ...
    synctex_class_p class;
}
~~~

Our current workaround is to rename that variable in our local copy of the code.

Could you please change that variable to another name, so that I can use new versions of synctex without having to adapt our local copy every time. `class_` or `nclass` (for node class) would be reasonable alternatives. If desired, I can provide a pull request since I've done the changes anyway.

**Comment by jlaurens** [comment]
Fixed in 1.21
class → class_

**Comment by timhoffm** [comment]
Thanks!
